namenode/main.go

```diff
@@ -307,21 +400,32 @@ func (server *NameNode) pickDatanodes(num uint) []string {
 			break
 		}
 		for endpoint, _ := range iter.Value().(map[string]bool) {
```

Comment:
This can be simplified as `for endpoint := range iter.Value().(map[string]bool)`. I got this from IDE.

Reply:
Oh I guess this is my bad, let me fix it. Sorry for neglecting an IDE warning.